l10n/en-US/viewer.properties

```
@@ -249,3 +249,10 @@ password_cancel=Cancel
 printing_not_supported=Warning: Printing is not fully supported by this browser.
 printing_not_ready=Warning: The PDF is not fully loaded for printing.
 web_fonts_disabled=Web fonts are disabled: unable to use embedded PDF fonts.
+
+# Editor
+editor_select=Enter in edit mode
+editor_select_label=Edit Mode
```

```suggestion
editor_none=Disable Annotation Editing
editor_none_label=Disable Editing
```

With similar changes in the `viewer.html` file.